tools/install_venv.py

```diff
@@ -1,223 +0,0 @@
-# Copyright 2011-2020 CERN for the benefit of the ATLAS collaboration.
```

Comment:
Note, this might still be mentioned in some documentations, need to remove it there.

Reply:
will have a look

Reply:
Actually, venv is not even mentioned in any documentation piece neither here nor in rucio-documentation – that we might want to change